frontend-react/src/components/FileHandlers/FileHandler.test.tsx

```diff
@@ -120,10 +120,24 @@ describe("FileHandler", () => {
             );
             expect(screen.getByText("Select data model")).toBeVisible();
         });
+
+        test.only("allows the user to upload and file and shows the success screen", async () => {
```

Comment:
Why do you need the test.only here and above? You want to skip the others?

Reply:
Yes, the others are broken. Instead of commenting out, I wanted to use the jest methods, `jest.skip` and `jest.only`